recipes/libconeangle/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "libdlfind" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 1f74754183432b06c5ada827f2cbf6f9d453c16a7638872db85fae164e88d128
+
+build:
+  skip: true  # [py<38 or python_impl == 'pypy']
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - python                                 # [build_platform != target_platform]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

Comment:
```suggestion
```
[Logs indicate](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=546080&view=logs&j=6f142865-96c3-535c-b7ea-873d86b887bd&t=22b0682d-ab9e-55d7-9c79-49f3c3ba4823&l=2804) that there are no links to libcxx. I don't think the cxx compiler is needed.